i18n/en.json

```diff
@@ -1352,10 +1352,18 @@
     "id": "api.incoming_webhook.disabled.app_error",
     "translation": "Incoming webhooks have been disabled by the system admin."
   },
+  {
+    "id": "api.incoming_webhook.invalid_signing_model.app_error",
+    "translation": "Invalid wehook authentication information."
+  },
   {
     "id": "api.incoming_webhook.invalid_username.app_error",
     "translation": "Invalid username."
   },
+  {
+    "id": "api.incoming_webhook.invalid_white_ip_list.app_error",
+    "translation": "The white ip list contains an invalid IP address or CIDR."
```

Comment:
`white ip list` sounds awkward -- should this be `ip whitelist`?, similarly for the `id`

Reply:
Thanks for point out the bad grammar, fixed all where it occurs.